src/Resources/skeleton/crud/test/Test.tpl.php

```diff
@@ -0,0 +1,134 @@
+<?= "<?php\n" ?>
+
+namespace <?= $namespace ?>;
+
+use Symfony\Bundle\FrameworkBundle\Test\WebTestCase;
+
+class <?= $class_name ?> extends WebTestCase<?= "\n" ?>
+{
+    public function testIndex()
+    {
+        $client = static::createClient();
+        $crawler = $client->request('GET', '<?= $route_path ?>/');
+
+        $this->assertSame(200, $client->getResponse()->getStatusCode());
+        $this->assertContains('<!DOCTYPE html>', $client->getResponse()->getContent());
+        $this->assertContains('<?= $entity_class_name ?> index', $crawler->filter('h1')->text());
+
+        $newLink = $crawler->filter('a:contains("Create new")')->eq(0)->link();
+
+        return $newLink;
+    }
+
+    /**
+     * @TODO implement new test
+     * @depends testIndex
```

Comment:
I'm not sure if this is a good idea. The tests should be isolated, and it's not like we *really* "need" the `testIndex` to be run first - we just need a value that it conveniently returns. I think we should remove this dependency and go directly to the index here then find that new link. This will make the `testNew()` tell a "complete story" without reading another test method.

Reply:
Tests are now isolated.